spring-cloud-stream-binder-rabbit-docs/src/main/asciidoc/overview.adoc

```diff
@@ -100,11 +100,33 @@ autoBindDlq::
   Whether to automatically declare the DLQ and bind it to the binder DLX.
 +
 Default: `false`.
+bindQueue::
```

Comment:
I wonder if we pursue shortened self-explained names.
As for me just `bind = true/false` is pretty clear.